### PR TITLE
ref #261: Configure DoctrineBundle

### PR DESCRIPTION
**Pull request by umulmrum**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | #261
| License       | MIT

### CODE REVIEW

File: app/config/config_dev.yml [hunk 1]
```diff
@@ -1,9 +1,6 @@
 imports:
    - { resource: config.yml }
 
-chameleon_system_debug:
```
Comment (UlrichKu) [inline app/config/config_dev.yml]: But `chameleon_system_debug: database_profiler: backtrace_enabled` is not added?

That is already false by default I guess?

File: app/config/config.yml [hunk 2]
```diff
@@ -17,3 +14,21 @@ chameleon_system_cms_text_field:
...
+    user:           '%database_user%'
+    password:       '%database_password%'
+    driver:         'pdo_mysql'
+    server_version: '5.7'
+    charset:        'utf8'
+    default_table_options:
+      charset: 'utf8'
+      collate: 'utf8_unicode_ci'
+    options:
+      1006: 1
```
Comment (UlrichKu) [inline app/config/config.yml]: What advantages has compression (by default)?

File: app/config/config_dev.yml (hunk 1 above)
Comment (umulmrum) [reply app/config/config_dev.yml]: Yes.

File: app/config/config.yml (hunk 2 above)
Comment (umulmrum) [reply app/config/config.yml]: Good catch, this wasn't intended.

**Pull request merged**